crates/pairwise_nostd_internal/src/apply_points.rs

```diff
@@ -170,3 +135,34 @@ pub fn apply_accum(
     }
     Ok(())
 }
+
+fn apply_accum_helper<const CROSS: bool>(
```

Comment:
Just trying to make sure I understand: you didn't actually change this function, right? You just relocated it?